Casks/virtual-desktop-streamer.rb

```diff
@@ -1,5 +1,5 @@
 cask "virtual-desktop-streamer" do
-  version "1.26.4"
+  version "2023.04.26"
```

Comment:
I think this is the right version.
```suggestion
  version "1.26.6"
```

Reply:
> I think this is the right version.

Yes I somehow typed in a version for another software. Oops.